Gemfile.lock

```diff
@@ -272,4 +272,4 @@ DEPENDENCIES
   whenever (~> 0.10.0)
 
 BUNDLED WITH
-   1.16.0
```

Comment:
We try and avoid committing changes to the Bundled With version.

Reply:
ok thanks